packages/go/dawgs/graph/node.go

```diff
@@ -238,12 +238,12 @@ func (s NodeSet) IDs() []ID {
 	return idList
 }
 
-// IDBitmap returns a new roaring64.Bitmap instance containing all Node ID values in this NodeSet.
```

Comment:
Should the const on line 30 be assigned to `math.MaxUint64`?

Reply:
Correct, updating.

Reply:
Chose `math.MaxInt64` since this is the upper limit of what can currently go into a given database.